docker/build_scripts/build_utils.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# Helper utilities for build
+
+PYTHON_DOWNLOAD_URL=https://www.python.org/ftp/python
+OPENSSL_DOWNLOAD_URL=http://www.openssl.org/source
+GET_PIP_URL=https://bootstrap.pypa.io/get-pip.py
+
+
+function check_var {
+    if [ -z "$1" ]; then
+        echo "required variable not defined"
+        exit 1
+    fi
+}
+
+
+function pyver_ge {
+    # Echo 1 if first python version is greater or equal to second
```

Comment:
Or even if we don't trust Guido's crystal ball, then this approach would work for us and is much cleaner: http://stackoverflow.com/a/15324047/1925449